streamingpro-it/src/test/resources/sql/simple/20_nesting_variables_across_cells_test.mlsql

```diff
@@ -0,0 +1,13 @@
+--%comparator=tech.mlsql.it.IgnoreResultComparator
+-- first cell
+
+set table_name="T_TableName" where scope="session";
+set ods_table_name="ODS_${table_name}" where scope="session";
+select "${ods_table_name}" as name as `${ods_table_name}`;
+
+-- second cell
+
+select name from `${ods_table_name}` as ccc;
+
+-- make sure status of  all models are success.
+!assert result ''':name=="ODS_T_TableName"'''  "all model status should be success";
```

Comment:
This line equals:

assert(:name=="ODS_T_Table_Name", "The variable ods_table_name should be ODS_T_TableName")

So it's better to change the description in the last part.

Reply:
1) remove the comment line `--make sure satus....`;
2) change the description `all model status should be success` which is not suite for your test.